packages/did-provider-key/src/index.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Provides `did:web` {@link @veramo/did-provider-key#KeyDIDProvider | identifier provider } for the
```

Comment:
This should reference did:key, not did:web
```suggestion
 * Provides `did:key` {@link @veramo/did-provider-key#KeyDIDProvider | identifier provider } for the
```